site/docs/types/0/0042-Software-Capabilities.md

```diff
@@ -53,6 +53,11 @@ In addition, it is possible to augment software capabilities with the following
 
 Defines the relationship between a `SoftwareCapability` and the `ITInfrastructure` asset that hosts it.
 
+## ProcessingState
+
+Defines a classification for a `SoftwareCapability`. It stores the last sync timestamp for critical elements.
+
+
 ??? deprecated "Deprecated types"
     - `SoftwareServerSupportedCapability` is deprecated in favor of `SupportedSoftwareCapability`.
```

Comment:
The diagram has the ProcessingState classification in gray and it should be in yellow

Reply:
I thought that classifications are in gray, as here : https://egeria-project.org/types/7/0790-Incomplete/?h=incom